plugins/alexa/src/main.ts

```diff
@@ -214,19 +261,47 @@ class AlexaPlugin extends AutoenableMixinProvider implements HttpRequestHandler,
         const json = JSON.parse(request.body);
         const { grant } = json.directive.payload;
         this.storageSettings.values.tokenInfo = grant;
-        this.getAccessToken();
+        this.accessToken = undefined;
+        
+        let accessToken = await this.getAccessToken().catch(reason => {
+            this.storageSettings.values.tokenInfo = undefined;
+            this.accessToken = undefined;
+
+            response.send(JSON.stringify({
+                "event": {
+                    "header": {
+                        "namespace": "Alexa.Authorization",
+                        "name": "ErrorResponse",
+                        "messageId": createMessageId(),
+                        "payloadVersion": "3"
+                    },
+                    "payload": {
+                        "type": "ACCEPT_GRANT_FAILED",
+                        "message": `Failed to handle the AcceptGrant directive because ${reason}`
+                    }
+                }
+            }));
+        });
 
-        response.send(JSON.stringify({
-            "event": {
-                "header": {
-                    "namespace": "Alexa.Authorization",
-                    "name": "AcceptGrant.Response",
-                    "messageId": createMessageId(),
-                    "payloadVersion": "3"
-                },
-                "payload": {}
+        if (accessToken !== undefined) {
+            try {
+                response.send(JSON.stringify({
```

Comment:
I don't think it's possible for this block to fail.

Reply:
This was failing during the debug process if the response didn’t come before the timeout.  Theoretically this is a debug problem, however it could occur in the wild.